### PR TITLE
fix cve

### PR DESCRIPTION
**Pull request by devinleee**
Signed-off-by: yuswift <yuswiftli@yunify.com>

### CODE REVIEW

**Comment by devinleee** [comment]
fix https://github.com/kubesphere/kubesphere/issues/3740#issuecomment-820956516

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o agent cmd/agent/main.
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM alpine:3.9
+FROM alpine:3.10.9
```
Comment (RolandMa1986) [inline Dockerfile]: Maybe we can use alpine:3.11, because most of the other images based on it.

Comment (devinleee) [reply Dockerfile]: done

**Pull request merged**